packages/react-components/react-datepicker/src/components/CalendarDay/CalendarDay.types.ts

```diff
@@ -143,34 +128,34 @@ export interface CalendarDayStyleProps extends CalendarDayGridStyleProps {
 /**
  * {@docCategory Calendar}
  */
-export interface CalendarDayStyles extends Partial<CalendarDayGridStyles> {
+export interface CalendarDayStyles {
```

Comment:
Shouldn't you still keep the `extends` here?

Reply:
It's not needed since it was unused as you can see in the deleted lines -153 to -174 in useCalendarDayStyles.ts